tests/e2e/default/contacts/edit.wdio-spec.js

```diff
@@ -0,0 +1,30 @@
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const utils = require('../../../utils');
+
+
+describe('Editing contacts with the default config. ', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const healthCenterPrimaryContact = userFactory.build({ place: healthCenter._id });
+  const docs = [...places, healthCenterPrimaryContact];
+
+  before(async () => {
+    await utils.saveDocs(docs);
+    await utils.createUsers([healthCenterPrimaryContact]);
```

Comment:
The user `healthCenterPrimaryContact` is created but is never used as a user. 
Do you really need to create a user?

Reply:
You are right @tatilepizs. Maybe to follow [this](https://github.com/medic/cht-core/issues/5704) issue, I can change the test to log in as an offline user and delete the primary contact. I may need to create another place with the primary contact to be deleted, because the contact deleted right now is related to the offline user. What do you think?

Reply:
That is a really good idea, and it is align with what we want in the future.
thanks @lorerod ! 😄